### ISSUE TITLE
Index Optimize - Index Type

### ISSUE THREAD

**Issue by jerkyflexoff** [olahallengren/sql-server-maintenance-solution#500]
Hi Ola,

Avid user of your code, wouldn't do without it.
Come across a problem recently where Geospatial Developers have required to maintain Spatial Indexes independently from the rest of the index types. 
We would like to continue to maintain but leave the spatial indexes for the developers application to manage.

We checked your great parameters under - SQL Server Index and Statistics Maintenance, Indexes. We could not identify a efficient way of doing this. Adding 1000+ spatial indexes as an exclusion is was not filling me with glory.

We checked your code and tried to remove the selection 4  from line, 1573, 1763 and 1764 - indexes.[type] IN(1,2,3,4,5,6,7) in [dbo].[IndexOptimize].

        when i.[type] = 1 then 'Clustered index'
        when i.[type] = 2 then 'Nonclustered unique index'
        when i.[type] = 3 then 'XML index'
        when i.[type] = 4 then 'Spatial index'
        when i.[type] = 5 then 'Clustered columnstore index'
        when i.[type] = 6 then 'Nonclustered columnstore index'
        when i.[type] = 7 then 'Nonclustered hash index'

Basically the jury is out on this and we need to do some more testing as I dont think it worked. 

The request from me is - To include index type inclusion\exclusion for the maintenance task. - [dbo].[IndexOptimize]. Please.

Thanks in advance .

Jake Williams
